systemtests/tests/autochanger/testrunner-label-release-autoselect

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+
+TestName="$(basename "$(pwd)")"
+export TestName
+
+#shellcheck source=./environment
+. ./environment
+. ./test-config
+
+#shellcheck source=../../scripts/functions
+. "${rscripts}"/functions
+
+start_test
+
+on_error() {
+  local lc="${BASH_COMMAND}"
+  echo "Error occurred in testrunner script [${lc}]"
+  export estat=1
+  exit 1
+}
+trap 'on_error' ERR
+
+echo "Set debug on the Tape-Storage daemon"
+tracefile="$(echo "setdebug level=200 trace=1 storage=Tape-0" | bin/bconsole | grep "^3000" | awk -F"tracefile=" '{print $2}' )"
+: > "${tracefile}" # clear tracefile
+
+echo "Label barcodes"
+for i in $(seq ${NUMBER_OF_POOLS}); do
+  pool=$(( i -1 )) #counts from 0
+  for j in $(seq ${NUMBER_OF_TAPES_PER_POOL}); do
+    slot=$(( j + pool * NUMBER_OF_TAPES_PER_POOL )) #counts from 1
+    echo "label barcodes slot=${slot} drive=0 pool=Full-${pool} storage=Tape-0 yes" | bin/bconsole | grep -E "(OK label|already exists)"
+  done
+done
+
+# create the test backup jobs
+rm -f bconsole_backup_jobs
+
+spooling="spooldata=yes"
+for i in $(seq ${NUMBER_OF_TEST_ROUNDS}); do
+  for j in $(seq ${NUMBER_OF_POOLS}); do
+    cat << EOF >> bconsole_backup_jobs
+run job=backup-bareos-fd level=Full storage=Tape-0 pool=Full-$(( j -1 )) ${spooling} yes
+status dir
+EOF
+    if [ $(( NUMBER_OF_SPOOLING_JOBS_PER_ROUND -j )) -le 0 ]; then
+      spooling=""
+    fi
+  done
+done
+
+# start all the jobs
+bin/bconsole < bconsole_backup_jobs
+
+testdevice0=tapedrive-0-.dev.tape.by-id.scsi-350223344ab000100-nst
+testdevice1=tapedrive-1-.dev.tape.by-id.scsi-350223344ab000200-nst
```

Comment:
The device names can easily change. These are just the devices mhvtl provides, but you can change the WWNs in mhvtl. Or maybe you run the tests with a real library, which will have different WWNs, too.
Also, people might have just one drive (esp. with a real library) so you cannot always rely on having two drives available - maybe we need to disable this (sub)test if there is only one drive available.

Reply:
you are right, I've changed the devices from hardcoded ones to being extracted from the config. I also changed it so it uses only one device.